.gitlab/llnl-ci-conda.yml

```diff
@@ -0,0 +1,117 @@
+stages:
+  - env-setup
+  - test
+  - cleanup
+
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SDK_WORK_DIR/conda || true
+    - chmod -fR 02770 $SDK_WORK_DIR/conda    || true
+
+conda-env-setup:
+  stage: env-setup
+  parallel:
+    matrix:
+      - HOST : [ "quartz", "ruby"]
+        RUNNER_TYPE: "shell"
+        # - HOST : [ "lassen" ]
+        #   ENVIRONMENT: [ "BATCH" ]
+        #   RUNNER_TYPE: "batch"
+  tags:
+    - $HOST
+    - $RUNNER_TYPE
+  script:
+    - mkdir -p ${SDK_WORK_DIR}/conda/
+    - cd ${SDK_WORK_DIR}/conda/
+    - eval "$(conda shell.posix hook)"
+    - conda create --prefix ./${HOST}-exaworks-sdk-venv python=3.7 -y
+    - conda env list
+    - conda activate ./${HOST}-exaworks-sdk-venv
+    - conda env list
+    - ls
+    - pwd 
+    - conda config --add channels conda-forge
+    - conda config --add channels lightsource2-tag 
+    - conda install swift-t
+    - conda install parsl
+    - conda install radical.pilot
+    - pip install pytest
+    - pip install pytest-tap
```

Comment:
schema of yml-file for environment allows to set pip-packages in it (btw, `pytest` is available in conda-forge)

Reply:
I added the requirement environment requirement files.

For the conda environment, I pip installed `pytest` and `pytest-tap`. Conda was taking forever installing `pytest` so I reasoned that using pip would speed up the environment setup process